pyiron_base/database/tables.py

```diff
@@ -77,3 +78,29 @@ def __init__(self, table_name, metadata):
             Column("id2", Integer),
             extend_existing=True,
         )
+
+
+class SimulationTable(PyironTable):
+    def __init__(self, table_name, metadata):
+        super().__init__(table_name, metadata)
+        self._table = Table(
+            table_name,
+            metadata,
+            Column("id", Integer, primary_key=True, autoincrement=True),
+            Column("parentid", Integer),
+            Column("masterid", Integer),
+            Column("projectpath", String(50)),
+            Column("project", String(255)),
+            Column("job", String(50)),
+            Column("subjob", String(255)),
+            Column("chemicalformula", String(30)),
```

Comment:
I don't know how exactly we can change this, but `chemicalformula`, `hamilton`, and `hamversion` are specific to atomistics jobs. One way can be to have separate tables for different pyiron modules (continuum, atomistics, etc), which I believe is not desirable. On the other hand we can keep the atomistic columns and add continuum/experimental related columns as well, then we would have a sparse table.

Reply:
Yes, they are definitely atomistic-specific. And in the case of the hamilton stuff they're not even well named any more. I would propose to stop using this table entirely and start using only the sparse table. I refactored the existing table out here for pedagogical reasons, but it also highlights how little this code belongs in `pyiron_base`!

I hadn't thought of including sub tables for each repo, but I'm open to it!